cs_unittest/TestModelLoading.cs

```diff
@@ -73,11 +73,11 @@ private void InternalTestModelRandomCorrupt(string modelFile)
                         Assert.IsTrue(false);
                     }
                 }
-                catch (VowpalWabbitException ex)
+                catch (VowpalWabbitException)
```

Comment:
can you remove this catch? the one below is enough. I think this was accidentally checked in while I was debugging.